pkg/services/sqlstore/migrations/correlations_mig.go

```diff
@@ -59,4 +59,8 @@ func addCorrelationsMigrations(mg *Migrator) {
 		"description": "description",
 		"config":      "config",
 	})
+
+	mg.AddMigration("add provisioning column", NewAddColumnMigration(correlationsV1, &Column{
```

Comment:
shouldn't this be `correlationsV2` ?

```suggestion
	mg.AddMigration("add provisioning column", NewAddColumnMigration(correlationsV2, &Column{
```

Reply:
Thanks for spotting this, I forgot to update after merging from `main`.